pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -65,19 +83,17 @@ type CiliumBGPPeeringPolicySpec struct {
 type CiliumBGPNeighborGracefulRestart struct {
 	// Enabled flag, when set enables graceful restart capability.
 	//
-	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Required
```

Comment:
Well, yeah I agree that we are now a bit inconsistent with regards to the `eBGPMultihopTTL`.
I remember we went this way (putting GR options into a struct) because `gracefulRestartRestartTimeSeconds` filed looked weird to us :)
Thoughts @YutaroHayakawa / @harsimran-pabla ?

Reply:
I think bool field for enabling/disabling graceful restart cap makes sense.

Right now, we are only exposing Restart Time, but this may change in future as we expose more configuration options described [here](https://github.com/osrg/gobgp/blob/master/api/gobgp.pb.go#L7553-L7563)

We should not tie in enabling/disabling to restart time - going back from it will be a breaking change in API.

Reply:
Good points @harsimran-pabla . At the same time I don't think that `eBGPMultihopTTL` would need its own struct and `enabled` flag similar to GR - as there are no other possible options.

Reply:
Agree to the conclusion of Pabla and Rastislav here 👍

Reply:
The inconsistency with `eBGPMultihopTTL` resolved in  https://github.com/cilium/cilium/pull/26040/commits/9958110ed9b17fc70180d6617ab2c34ef29a1b10